### PR TITLE
[14.0][MIG] fieldservice_sale

### PR DESCRIPTION
**Pull request by Freni-OSI**
Depends on #763

### CODE REVIEW

File: fieldservice_sale/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    service_type = fields.Selection(
+        selection_add=[
+            ("field", "Field Service Orders"),
```
Comment (britoederr) [inline fieldservice_sale/models/product_template.py:13]: Missing ondelete attribute:
ondelete={'field':'set default'}

File: fieldservice_sale/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    qty_delivered_method = fields.Selection(
+        selection_add=[("field_service", "Field Service Order")]
```
Comment (britoederr) [inline fieldservice_sale/models/sale_order_line.py]: Missing ondelete attribute:
ondelete={'field_service':'set default'}

**Review by britoederr — CHANGES_REQUESTED** [review]

File: fieldservice_sale/__manifest__.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "depends": [
         "fieldservice",
         "sale_management",
-        "fieldservice_account",
```
Comment (hparfr) [inline fieldservice_sale/__manifest__.py]: Why this change ?

**Review by hparfr — CHANGES_REQUESTED** [review]

File: fieldservice_sale/__manifest__.py (hunk 3 above)
Comment (Freni-OSI) [reply fieldservice_sale/__manifest__.py]: We have used customer_id which is introduced in fieldservice_account_analytic.

File: fieldservice_sale/models/sale_order_line.py (hunk 2 above)
Comment (Freni-OSI) [reply fieldservice_sale/models/sale_order_line.py]: Fixed. Thanks

**Comment by Freni-OSI** [comment]
Needs Review
CC: @patrickrwilson @smangukiya

File: fieldservice_sale/__manifest__.py (hunk 3 above)
Comment (hparfr) [reply fieldservice_sale/__manifest__.py]: Please remove the introduction of customer_id.

**Review by patrickrwilson — APPROVED** [review]
Tested and works great. LGTM

**Comment by patrickrwilson** [comment]
@hparfr @britoederr do you mind reviewing this now that changes were made?

**Comment by hparfr** [comment]
This PR adds a field and a dependency to fieldservice_account_analytic.  I still request to not introduce this change.

**Comment by Freni-OSI** [comment]
> This PR adds a field and a dependency to fieldservice_account_analytic. I still request to not introduce this change.

Removed dependency and field.

**Review by hparfr — APPROVED** [review]
LGTM

**Comment by hparfr** [comment]
/ocabot merge nobump

thanks @Freni-OSI
